src/components/pages/get-a-quote/form.svelte

```diff
@@ -53,6 +53,10 @@
     lastChangedInput = value.detail;
   };
 
+  $: if (lastChangedInput !== 'budget' && budget.length) {
+    lastChangedInput = 'budget';
+  }
+
```

Comment:
With this piece of code, we're overriding character changes.
For example, if we choose a budget and then add an attachment, the character will still be the budget related one, and won't change to the attachment one.

Since it looks like it is here because changing the option in the radio button doesn't update the character, we can track the onchange event on the Contact form and dispatch a change event on the Radio component to achieve the same result. That way it doesn't overwrite the other character changes and also works if the user uses the keyboard to change between budget options

Reply:
True, fixed